### PR TITLE
Tournament lances

### PR DESCRIPTION
**Pull request by unid15**
- Added tournament lances. They are very heavy weapons that do 3 melee damage and significantly slow you down when carried
- Tournament lances can be couched (lowered in front of you). When couching a lance, you have to move very fast without turning, or you'll lose control of the lance and raise it. There's a 5 second cooldown after the lance is raised.
- Damage done by the lance is 3*(turfs traveled while couching). To deliver the damage, you must either click on your foe (which is pretty hard at high speeds) or just collide with them. With enough force, your victim will be weakened or even thrown back!
- Unfortunately, super high speeds don't affect the damage. 
- Tournament lances can be found in xenoarch

---

Other changes:
- SLOWDOWN_WHEN_CARRIED item flag
- Includes #8624 

---

Notes:
- Couching works by atom locking an invisible trigger to the player. Whenever the trigger is moved, it checks for all living mobs in the new turf and impales one of them.
- DELIVERED COUCHED LANCE DAMAGE is shown when delivering couched lance damage (M&B reference)

### CODE REVIEW

**Comment by TheFriggenSawbones** [comment]
:+1: :+1: :+1:  :100:   :joy: :ok_hand:

**Comment by gbasood** [comment]
> xenoarch lances

So what you're telling me is there was a civilization that jousted on asteroids for funsies

**Comment by ihadtoregisterforthis** [comment]
Better than a civilization that jousted on the maintenance tunnels on a space station.

**Comment by unid15** [comment]
Well that civilization had houses, temples and wars so jousting tournaments aren't that far-fetched

**Comment by cowboybuttsdrivemenuts** [comment]
Holy fuck yes

**Comment by gbasood** [comment]
So wait if this includes #8624 shouldn't the PR conflict or look different or something

**Comment by unid15** [comment]
it's the same commit so I don't see why it would conflict

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -4,7 +4,10 @@
 	var/w_type = NOT_RECYCLABLE  // Waste category for sorters. See setup.dm
 
 	layer = 3
-	var/last_move = null
+
+	var/last_move = null //Direction in which this atom last moved
+	var/last_moved = 0   //world.time when this atom last moved
+
 	var/anchored = 0
```
Comment (gbasood) [inline code/game/atoms_movable.dm]: oh boy look at these duplicate lines being deleted and re-added

**Comment by gbasood** [comment]
For the sake of sanity you might want to:
pull changes
rebase maybe
reset atom_movables.dm to Bleeding-Edge

If it's got changes to that file you need to keep I can review shit and make sure it won't merge badly

**Comment by unid15** [comment]
Done!

**Pull request merged**